styles/designs/webview/parts/_exercise-components.scss

```diff
@@ -111,6 +111,46 @@ $Exercise__Problem__MCAnswer__Pre: (
   )
 );
 
+
+$InjectedExercise__Question__MultiChoiceAnswers: (
+  _name: "InjectedQuestionAnswers",
+  _subselector: " ol[data-type=question-answers][type=a]",
```

Comment:
Are there other types?

Reply:
No, only `type a` at the moment. Right now there is a hard requirement that these lists are `type a` [here](https://github.com/openstax/cookbook/blob/b74ed39cfb29d45b2e8fe38fbc8e625d9bdab3e7/lib/kitchen/directions/bake_injected_exercise/bake_injected_exercise_question.rb#L44) when solutions are being moved around. I mostly added this to be explicit about which type of list to target.